src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,83 @@
+package duke;
+
+import java.util.ArrayList;
+
+/**
+ * The TaskList contains the list of tasks and operations to modify the list.
+ */
+public class TaskList {
+    private ArrayList<Task> taskList;
+
+    /**
+     * Constructor for a TaskList object with an existing list.
+     * @param taskList An arraylist of Tasks.
```

Comment:
Perhaps you should leave a line between your description and your @param line?

Reply:
Alright thanks